Sources/SimpleSalesforceSDK/Models/URLRequestBuilder.swift

```diff
@@ -0,0 +1,48 @@
+import Foundation
+
+public struct URLRequestConfig {
+  var url: URL
+  var params: Data?
+  var bearerToken: String?
+  var httpMethod = HTTPMethod.get
+  var contentType = CONTENTTYPE.urlEncoded
+
+  enum HTTPMethod: String {
+    case get = "GET"
+    case head = "HEAD"
+    case post = "POST"
+    case put = "PUT"
+    case delete = "DELETE"
+    case connect = "CONNECT"
+    case options = "OPTIONS"
+    case trace = "TRACE"
+    case patch = "PATCH"
+  }
+  
+  enum CONTENTTYPE: String {
+    case json = "application/json"
+    case urlEncoded = "application/x-www-form-urlencoded"
+  }
+}
+
+public struct URLRequestBuilder {
+  ///  Creates URL for updateData api
```

Comment:
Wrong comment and we can just call 
```suggestion
public struct RequestBuilder {
  ///  Builds requests based on provided config
```